### ISSUE TITLE
[Fixed] Brackets Highlight not working

### ISSUE THREAD

**Issue by Suraj-Tiwari** [sectasy0/fleet_one_dark#3]
In Fleet Provided Themes brackets highlight is working for all `{}`, `()`, `[]`  but not in one dark theme.

Default Theme:
<img width="147" alt="Screenshot 2023-08-26 at 2 08 23 AM" src="https://github.com/sectasy0/fleet_one_dark/assets/22210490/723d2299-63ea-4639-b187-07ea0843a4df">

One Dark Theme:
<img width="147" alt="Screenshot 2023-08-26 at 2 08 51 AM" src="https://github.com/sectasy0/fleet_one_dark/assets/22210490/a9badfa5-6b77-49c1-9bc5-4ca6f2d17b75">

TLDR; i don't have enough knowledge of themes to identify and fix this.

**Comment by sectasy0** [comment]
Hi, are you referring to the background for the signs? In One Dark these characters do not have any fixed background, it is a standard behavior. If you want I can help you set

**Comment by Suraj-Tiwari** [comment]
Yeah that'll be great, i need a way to get them highlighted because it causes issues when it's nested.
Btw, thanks for this cool theme.

**Comment by sectasy0** [comment]
> Yeah that'll be great, I need a way to get them highlighted because it causes issues when it's nested. Btw, thanks for this cool theme.

I figured out how to add this highlight, you just have to put it before "comment". I will push a commit later

```json
"editor.brace.highlighted": {
    "bgColor": "#515a6b", 
    "layer": 300
 },
```

**Comment by Suraj-Tiwari** [comment]
Thanks Bro, it works now.
